src/amo/components/SearchForm.js

```diff
@@ -35,8 +36,8 @@ export class SearchFormBase extends React.Component {
         <label className="visually-hidden" htmlFor="q">{_('Search')}</label>
         <input ref={(ref) => { this.searchQuery = ref; }} type="search" name="q"
                placeholder={_('Search extensions and themes')}
-               defaultValue={query} />
-        <button className="button button-middle" type="submit" title="Enter"
+               defaultValue={query} className="SearchForm-query" />
+        <button className="visually-hidden" type="submit" title="Enter"
```

Comment:
We should consider the implications of the hidden submit. It might not matter too much on Mobile, but something we could add to the list for testing if we get an accessibility review once we have this up on -dev.